client/src/components/Reviews/ClickTracker.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import API from './ReviewsAPIUtils';
+
+const ClickTracker = (WrappedComponent) => {
+
```

Comment:
this looks amazing! as a side note, i'm wondering what the difference is between making the HOC return a class vs functional component. i did the former, but prob didn't need to since almost all of my components are functional...need to learn more.

Reply:
Probably the same as with normal components, you would use a class if it needs to have state?